src/breeze/components/breeze/sidebar_links.cr

```diff
@@ -1,19 +1,32 @@
 class Breeze::SidebarLinks < BreezeComponent
   needs context : HTTP::Server::Context
 
-  # NOTE: https://github.com/luckyframework/lucky/issues/1243
-  # Habitat.create do
-  #   setting links : Hash(String, BreezeAction.class) = {
-  #     "Requests" => Breeze::Requests::Index,
-  #     "Queries"  => Breeze::Queries::Index,
-  #   }
-  # end
+  Habitat.create do
+    setting links : Array(Proc(Breeze::SidebarLinks, Nil)) = [
+      ->(breeze : Breeze::SidebarLinks) {
```

Comment:
If I change this to `->(breeze : BreezeComponent)`, I actually get a different error...

```
web          | In lib/breeze/src/breeze/components/breeze/sidebar_links.cr:7:59
web          |
web          |  7 | breeze.mount(Breeze::SidebarLink, context: breeze.context, link_text: "Requests", link_to: Breeze::Requests::Index)
web          |                                                        ^------
web          | Error: undefined method 'context' for RowArrow (compile-time type is BreezeComponent+)
web          |
```

Obviously I'm not calling `RowArrow` anywhere here, but Crystal seems to think since `RowArrow` is also a `BreezeComponent`, then it needs to be a viable use here...

Reply:
This is working how I wanted now.